packages/yuudachi/locales/en-US/translation.json

```diff
@@ -39,7 +39,7 @@
 		"mod": {
 			"common": {
 				"errors": {
-					"max_length_reason": "Maximum length of `1900` for reason exceeded.",
+					"max_length_reason": "Maximum length of `500` for reason exceeded.",
```

Comment:
Wondering if this should be a constant somewhere? Not entirely sure if this number has meaning, but it might be useful to tie it to where it's being used.

Reply:
Will, it actually is, I just forgot to push the commit, OMG I'm so stupid

Reply:
![image](https://user-images.githubusercontent.com/67063134/192178027-aebb2a04-aefa-4aa4-90ac-1ada79be2c13.png)
LOL

Reply:
Fixed in ffefa4c

Reply:
That commit is not on this branch?

Reply:
LGTM once this string uses the constant!